### ISSUE TITLE
Struggling to install/run Aider on Windows

### ISSUE THREAD

**Issue by tdobson** [Aider-AI/aider#1507]
### Issue

Me and my friend @jkuriata are struggling to install Aider on Windows with VS Code. I'm aware of the vscode plugin, it's more just the vanilla aider I'm struggling with. We're trying to install from pip (and ultimately pipx).

The main problem that seems to be occurring is when it runs, it starts, but then at the moment where we expect it to drop into interactive mode, it sort of drops us back into powershell - like this:

PS C:\Users\Judyt\OneDrive\Documents\tutorial\myapp> pipx list
venvs are in C:\Users\Judyt\pipx\venvs
apps are exposed on your $PATH at C:\Users\Judyt\.local\bin
manual pages are exposed at C:\Users\Judyt\.local\share\man
   package aider-chat 0.56.0, installed using Python 3.12.6
    - aider.exe
PS C:\Users\Judyt\OneDrive\Documents\tutorial\myapp> Set-ExecutionPolicy RemoteSigned -Scope CurrentUser
PS C:\Users\Judyt\OneDrive\Documents\tutorial\myapp> aider
─────────────────────────────────────────────────────────────────────────────────────────
Aider v0.56.0
Model: deepseek/deepseek-coder with diff edit format, prompt cache, infinite output      
Git repo: .git with 16 files
Repo-map: using 1024 tokens, files refresh
PS C:\Users\Judyt\OneDrive\Documents\tutorial\myapp>

I'm put a lot of braincells to it, but I'm struggling to work it out. 

I did use this - but it hasn't created joy.
https://aider.chat/docs/troubleshooting/aider-not-found.html

Here's some of things we've tried

https://pastebin.com/7HxyhcWA

### Version and model info

Windows 11 Home
Aider: latest from pip
Python from Windows store

**Comment by fry69** [comment]
Not a Windows expert but I know that OneDrive and other auto-synced folders are a bad place for installing Python and git stuff:
- permissions get handled strangely in those folders
- git freaks out if something in its `.git` folder is getting inconsistent (e.g. through syncing/renaming/turning a file in to placeholder to save disk space, as seen on Apple iCloud Drive).

**Comment by paul-gauthier** [comment]
How did you install aider?

This doc might be helpful:

https://aider.chat/docs/install/pipx.html

**Comment by tdobson** [comment]
Hey @paul-gauthier , @fry69  - I've got it working, but it wasn't what I expected.

I used the pipx installer and that worked nicely.

The problem was (to my surprise) in the configuration file - and specifically because I'd specified to lint (but possibly not specified a linter?)

Here's the config file that caused the problem 
https://pastebin.com/raw/buAvnNWs

I removed 
lint: true

to make it work(?)

I think this can be closed from *my* point of view now - but if anyone else can replicate this, I'd be very interested.

It wasn't obvious that aider closing without error was caused by this. 

As always, thanks for this wonderful tool that has improved my life.
